lib/prerequisite.js

```diff
@@ -1,13 +1,49 @@
 'use strict';
 const execa = require('execa');
 const Listr = require('listr');
+const pTimeout = require('p-timeout');
 const version = require('./version');
 
 module.exports = (input, pkg, opts) => {
 	let newVersion = null;
 	let tagPrefix = 'v';
 
 	const tasks = [
+		{
+			title: 'Ping npm registry',
+			task: () => pTimeout(execa.stdout('npm', ['ping'])
+				.catch(() => {
```

Comment:
Why the `.catch` here? I think it could be removed.

Reply:
`npm ping` does wait too long, AFAIK. But we can't say for sure it will every time, or in future versions, maybe it's a quirk.

It took long when my PC's network card was disabled. But it might stderr before 6 seconds in case DNS server is not responding, or one of the gateways are down etc.

Reply:
You're right, missed your second point :).